indigo/commands.sbt

```diff
@@ -102,10 +104,7 @@ addCommandAlias(
 )
 addCommandAlias(
   "localPublishIndigo",
-  applyCommand(
-    coreProjects.filterNot(name => name == "sandbox" || name == "perf"),
```

Comment:
I think this is no longer needed as the `neverPublish` settings in the `build.sbt` file should take care of not publishing artefacts for these projects.

Reply:
actually the current implementation is still trying to publish them for some reason 🤔  
at least according to the logs. Looking into it

Reply:
I just noticed that (might be an older version of this PR) a bunch of modules that should not be published (because they aren't artefacts) were published, like the benchmarks project.

Final list in you local ivy should be indigo, extras, json, and the plugin artefacts (indigo-plugin 2.12 + 2.13, mill-indigo, sbt-indigo).

Reply:
oh yeah, I think it's because I had `skip /  publish := true`, instead of `publish / skip : true`. I've double-checked now and I can see the correct list is being published. (Indigo is being published twice, adding the `neverPublish` settings to the aggregate should fix this)